File: src/main/webapp/js/service/bbsService.js
//服务层
app.service('bbsService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../bbs/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../bbs/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../bbs/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../bbs/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../bbs/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../bbs/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../bbs/search.do?page='+page+"&rows="+rows, searchEntity);
	}    
	
	//查找主题帖及回复
	this.findTopAndReplys=function(uuid){
		return $http.get('../bbs/findTopAndReplys.do?uuid='+uuid);
	}   
});

File: src/main/java/com/cf/service/BbsService.java
package com.cf.service;
import java.util.List;

import com.cf.pojo.TbBbs;
import com.cf.pojo.group.BlogReplys;
import com.cf.pojo.result.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface BbsService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbBbs> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbBbs bbs);
	
	
	/**
	 * 修改
	 */
	public void update(TbBbs bbs);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbBbs findOne(String id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(String [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbBbs bbs, int pageNum,int pageSize);
	
	/**
	 * 查找主题贴及回复
	 * @param uuid
	 * @return
	 */
	public BlogReplys findBlogReplys(String uuid);
}
